recipe/meta.yaml

```diff
@@ -1,91 +1,122 @@
-{% set version = "1.9.1" %}
+{% set version = "1.9.6" %}
+{% set name = "geoviews" %}
 
 package:
-  name: geoviews-core
+  name: {{ name }}-split
   version: {{ version }}
 
 source:
-  url: https://github.com/holoviz/geoviews/archive/v{{ version }}.tar.gz
-  sha256: 458bbe40b4bbf7bba5ef7b99d89a28618c024102545bc03687b6659368b335ed
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e2ab66626fb57434ede7670ed8b2b2e8331919dbec78edb4d1d1646dfa0c73e7
 
 build:
   number: 0
+  #skip s390x as lack of nodejs and cartopy dependencies
+  skip: True  # [py<37 or s390x]
 
 outputs:
-  - name: geoviews-core
-    script: build_base.sh
+  - name: {{ name }}-core
+    script: build_base.sh # [not win]
+    script: build_base.bat # [win]
     build:
       noarch: python
       entry_points:
         - geoviews = geoviews.__main__:main
     requirements:
       run_constrained:
-        - geoviews {{ version }}
+        - {{ name }} {{ version }}
       host:
-        - python >=3.6
+        - python
         - pip
-        - param >=1.9.0
-        # These are also needed at build time and declared in the pyproject.toml
-        - pyct >=0.4.4
-        - bokeh >=2.3.0,<2.4.0
-        - nodejs >=10.13.0
+        - setuptools
         - wheel
-        - cartopy >=0.18.0
-        - setuptools >=30.3.0
+        - packaging
+        - param 1.9.0
+        - pyct 0.5.0
+        - bokeh 2.4.3
+        - nodejs 16.13.1
+        - cartopy 0.21.1
+        - pyviz_comms 0.7.2
       run:
-        - python >=3.6
-        - bokeh >=2.3.0,<2.4.0
+        - python
+        - bokeh >=2.4.0,<2.5.0
         - cartopy >=0.18.0
         - holoviews >=1.14.2
-        - numpy  >=1.0
+        - numpy >=1.0
         - param >=1.9.3
+        - pykdtree
     test:
       requires:
         - pip
       commands:
         - pip check
       imports:
-        - geoviews
+        - {{ name }}
 
-  - name: geoviews
+  - name: {{ name }}
     build:
       noarch: python
     requirements:
-      - python >=3.6
-      - bokeh >=2.3.0,<2.4.0
-      - cartopy >=0.18.0
-      - holoviews >=1.14.2
-      - numpy >=1.0
-      - param >=1.9.3,<2.0
-      - {{ pin_subpackage('geoviews-core', max_pin="x.x.x.x") }}
-      - datashader
-      - geopandas
-      - netcdf4
-      - pandas
-      - pyct
-      - xarray
+      host:
+        - python
+        - pip
+        - setuptools
+        - wheel
+      run:
+        - python
+        - bokeh >=2.4.0,<2.5.0
+        - cartopy >=0.18.0
+        - holoviews >=1.14.2
+        - numpy >=1.0
+        - param >=1.9.3,<2.0
+        - {{ pin_subpackage('geoviews-core', exact=True) }}
```

Comment:
Do we need to explicitly re-define some run-time dependencies here which were already mentioned as run deps of geoviews-core, since they should be implicitly added by geoviews-core anyway?

Reply:
removed